src/core/interpreter_polyfill.h

```diff
@@ -0,0 +1,90 @@
+// Copyright 2022, DragonflyDB authors.  All rights reserved.
+// See LICENSE for licensing terms.
+//
+// This header contains implementations of deprecated, removed or renamed lua functions.
+
+#pragma once
+
+extern "C" {
+#include <lauxlib.h>
+#include <lua.h>
+#include <lualib.h>
+
+// TODO: Fix checktab
+#define aux_getn(L, n, w) (luaL_len(L, n))
```

Comment:
I have no clue what this is and in what cases its error is better than the errors thrown by the raw length function. 

https://github.com/lua/lua/blob/cf08915d62e338c987b71c078b148490510e9fe7/ltablib.c#L46

Reply:
The solutions are:
- copy the checktab code with its dependencies 
- check manually for the table type (which excludes metatable objects that act like tables?)

I chose the second because of simplicity